box.json

```diff
@@ -1,5 +1,4 @@
 {
-    "chmod": "0755",
     "directories": [
```

Comment:
might be worth checking but you can move the current `files` to `files-bin` and remove completely `directories` and `finder`

Reply:
Thanks, I'll try it out